### PR TITLE
Bootstrap iife exercise

### PR DESCRIPTION
**Pull request by NewEvolution**
Quick exercise to explore some of the many options offered by Bootstrap and incorporating form data acquisition & IIFE private data storage.  Featuring both form & table styling and relying on the grid system to make the page responsive.

### CODE REVIEW

**Comment by JoeShep** [comment]
:+1:

File: single-page-applications/exercises/SP_JS_BOOTSTRAP_IIFE.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Sandwich Maker
```
Comment (chortlehoort) [inline single-page-applications/exercises/SP_JS_BOOTSTRAP_IIFE.md]: Change title

File: single-page-applications/exercises/SP_JS_BOOTSTRAP_IIFE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
++ a form for user information submission
++ the ability to select whether the donation will be a single amount or per lap
++ a table to display each donor's submission information
++ a button labeled Donate to process the submission
++ a button labeled Cancel to clear the submission form
+
+The donor information should be stored as JS objects that contain their content type as keys and the submitted information as the value. `{name: "John Donut", pledge: 70}`
+
+All donor information objects should be stored in a private array
+
```
Comment (chortlehoort) [inline single-page-applications/exercises/SP_JS_BOOTSTRAP_IIFE.md]: ... in an IIFE...

**Comment by chortlehoort** [comment]
Just a few small additions, then go ahead and merge it

**Pull request merged**